tests/testthat/test-get_modelmatrix.R

```diff
@@ -59,7 +57,11 @@ test_that("get_modelmatrix - iv_robust", {
 # ====================================================================
 
 test_that("get_modelmatrix - ivreg", {
+  skip_if(getRversion() < "4.0.0")
+  skip_if_not_or_load_if_installed("ivreg")
   data(Kmenta, package = "ivreg")
+
+  set.seed(15)
```

Comment:
Does this seed lead to the failing test on old-rel?

Reply:
No, I added it to fix the failing test, but it didn't do anything. No clue why unloading AER namespace leads these tests to fail.

Reply:
AER includes the "predecessor" of `ivreg`, maybe `AER::ivreg()` and `ivreg::ivreg()` give slightly different results on R 4.0? Maybe try `ivreg::ivreg()`, and see if this works (or fails only on one R version, and if so, skip all until oldrel (and older)?)

Reply:
I just skipped the test in question on older R version.